app-server/app/src/test/scala/organization/OrganizationSpec.scala

```diff
@@ -14,15 +23,133 @@ class OrganizationSpec extends WordSpec
     with DBSpec {
   implicit val ec = system.dispatcher
   implicit def database = db
+  implicit def default(implicit system: ActorSystem) = RouteTestTimeout(DurationInt(20).second)
+  val now = new java.util.Date()
+  val cal = java.util.Calendar.getInstance();
+  cal.setTime(now);
+  cal.add(java.util.Calendar.DATE, 1)
+  val tomorrow = cal.getTime()
+  val claims = Map(
+    "sub" -> JsString("Default"),
+    "name" -> JsString("Default User")
+  )
+  val token = JsonWebToken(claims, now.toInstant(), tomorrow.toInstant())
+  val encodedToken = JsonWebToken.write(token: JsonWebToken, auth0Secret)
+  val authorization = Authorization(OAuth2BearerToken(encodedToken))
+
+  "/api/organizations" should {
+    "require authentication" in {
+      Get("/api/organizations") ~> organizationRoutes ~> check {
+        rejection
+      }
+    }
 
-  "The organization service" should {
-    "return a list of organizations" in {
-      val routes = organizationRoutes
-      // TODO: implement tests once we've figured out how to mock JWT tokens for
-      //       endpoints that require authentication
-      // Get("/api/organizations") ~> routes ~> check {
-      //   responseAs[Seq[OrganizationsRow]]
-      // }
+    "return a paginated list of organizations" in {
+      Get("/api/organizations")
+        .addHeader(authorization) ~> organizationRoutes ~> check {
+        responseAs[PaginatedResponse[OrganizationsRow]]
+      }
+    }
+    "allow creation of new organizations" in {
+      val newOrg = OrganizationsRowCreate("Test Organization")
+      // val marshalledOrg = marshal(newOrg)
+      Post("/api/organizations")
+        .withHeadersAndEntity(
+        List(authorization),
+        HttpEntity(ContentTypes.`application/json`, newOrg.toJson(organizationsRowCreateFormat).toString())
+      ) ~> organizationRoutes ~> check {
+        responseAs[OrganizationsRow]
+      }
+    }
+  }
+
+  "/api/organizations/{uuid}" should {
+    "return an organization" in {
+      Get("/api/organizations")
+        .addHeader(authorization) ~> organizationRoutes ~> check {
+        val orgs = responseAs[PaginatedResponse[OrganizationsRow]]
+        val orgId = orgs.results.head.id
+
+        Get(s"/api/organizations/$orgId")
+          .addHeader(
+          RawHeader("Authorization", s"Bearer $encodedToken")
+        ) ~> organizationRoutes ~> check {
+          responseAs[OrganizationsRow]
+        }
+      }
+    }
+
+    "return a 404 for non-existent organizations" in {
+      val orgUUID = java.util.UUID.randomUUID()
+      Get(s"/api/organizations/$orgUUID")
+        .addHeader(RawHeader("Authorization", s"Bearer $encodedToken")) ~> organizationRoutes ~> check {
```

Comment:
There isn't - I figured out the better way to do it (using the Authentication header) and forgot to go back and change this one.